doc/man3/OPENSSL_malloc.pod

```diff
@@ -137,22 +124,7 @@ return a pointer to allocated memory or NULL on error.
 CRYPTO_set_mem_functions() returns 1 on success or 0 on failure (almost
 always because allocations have already happened).
 
-CRYPTO_mem_leaks(), CRYPTO_mem_leaks_fp(), CRYPTO_mem_leaks_cb(),
-CRYPTO_set_mem_debug(), and CRYPTO_mem_ctrl() are deprecated and are no-ops that
-always return -1.
-OPENSSL_mem_debug_push(), OPENSSL_mem_debug_pop(),
-CRYPTO_mem_debug_push(), and CRYPTO_mem_debug_pop()
-are deprecated and are no-ops that always return 0.
-
-=head1 HISTORY
-
-OPENSSL_mem_debug_push(), OPENSSL_mem_debug_pop(),
-CRYPTO_mem_debug_push(), CRYPTO_mem_debug_pop(),
-CRYPTO_mem_leaks(), CRYPTO_mem_leaks_fp(),
-CRYPTO_mem_leaks_cb(), CRYPTO_set_mem_debug(), CRYPTO_mem_ctrl()
-were deprecated in OpenSSL 3.0.
-The memory-leak checking has been deprecated in OpenSSL 3.0 in favor of
-clang's memory and leak sanitizer.
+CRYPTO_mem_ctrl() is deprecated it is a no-op that always returns -1.
```

Comment:
sigh this makes no sense.. why not just return CRYPTO_MEM_CHECK_OFF (aka 0) as one of the predefined values this function would historically return.

(of course no point in introducing an incompatibility here)

Reply:
OpenSSL has lots of these kinds of inconsistencies, returning 0/1 1/-1, 0/1/-1, etc.  It (used to?) drive @davidben up the wall :)